ttnn/cpp/ttnn/operations/experimental/ccl/reduce_scatter_minimal_async/device/reduce_scatter_minimal_async_program.cpp

```diff
@@ -2013,3 +2089,357 @@ tt::tt_metal::operation::ProgramWithCallbacks line_reduce_scatter_minimal_async_
     return {.program = std::move(program), .override_runtime_arguments_callback = override_runtime_arguments_callback};
 }
 }  // namespace ttnn
+
+// Implementations for the TMP namespace - wrappers to ttnn namespace functions
+namespace ttnn::operations::experimental::ccl::reduce_scatter_minimal_async_detail {
```

Comment:
```suggestion
namespace ttnn::operations::experimental::ccl::reduce_scatter_minimal_async_::detail {
```
Did you mean a nested namespace `detail` here?